### PR TITLE
Update SOLUP Token

### PR DESCRIPTION
**Pull request by deeptext2**
## I agree to not ping *anybody* on Discord/Twitter/email about this pull request. Instead I will inquire by posting a new comment in the pull request if needed.
----
PRs are reviewed in bulk and and can take up to **two weeks** to be merged.  

**Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR. 
DON'T modify any other token on the list.

At minimum each entry should have
* Token Address: 
* Token Name: 
* Token Symbol: 
* Logo: (logo should be uploaded under assets/mainnet/<mint address>/*.<png/svg>)
* Link to the official homepage of token:
* Coingecko ID if available (https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id_):

### CODE REVIEW

File: src/tokens/solana.tokenlist.json [hunk 1]
```diff
@@ -4068,6 +4068,23 @@
         "coingeckoId": "funfair"
       }
     },
+    {
+      "chainId": 101,
+      "address": "CnGUfvi9FxiRPuaBXpYmaXEwBjj5X6kwNJB2Cba5TiQp",
+      "symbol": "SOLUP",
+      "name": "SolanaUP",
+      "decimals": 8,
+      "logoURI": "https://raw.githubusercontent.com/deeptext2/solup/main/solup-green.svg",
```
Comment (bartosz-lipinski) [inline src/tokens/solana.tokenlist.json:4077]: please store logo in this repo

Comment (deeptext2) [reply src/tokens/solana.tokenlist.json:4094]: Thank you, We have updated the logo.

Comment (bartosz-lipinski) [reply src/tokens/solana.tokenlist.json:4094]: please add only one token to the list

Comment (deeptext2) [reply src/tokens/solana.tokenlist.json:4094]: Sure, We add only SOLUP token, Please check the last commit

**Comment by bartosz-lipinski** [comment]
Please dont use Solana in token name.

**Pull request closed without merging**